dev-site-config/navigation.config.js

```diff
@@ -17,9 +29,12 @@ const navConfig = {
         devTools: true,
       },
     }, {
-      path: '/contributing',
-      text: 'Contributing',
-      pageTypes: ['contributing'],
+      path: '/dev_environment',
+      text: 'Dev Environment',
```

Comment:
Looking at the deployment, I think the `Docker Development Env` page would makes sense to be groups under About > Contributing, potentially after Development Workflow

![Screen Shot 2020-03-31 at 9 43 55 AM](https://user-images.githubusercontent.com/14099737/78039787-2c7b6780-7334-11ea-92ee-7008962eb17e.png)

Reply:
I also think the Webpack Information should be listed under guides, `Guides > Bundle Assets with Webpack`. I think we eventually will need to add more information here (maybe) for setting up webpack vs what configs we are using/expecting